python2_requirements.txt

```diff
@@ -6,13 +6,15 @@ contextlib2==0.5.5
 entrypoints==0.3
 enum34==1.1.6
 flake8==3.7.7
+funcsigs==1.0.2
 functools32==3.2.3.post2
 futures==3.2.0
 identify==1.4.3
 importlib-metadata==0.13
 importlib-resources==1.0.2
 isort==4.3.20
 mccabe==0.6.1
+mock==3.0.5
```

Comment:
This is incorporated into the unittest module in python 3, but in python 2 is external.